src/rabbit_stomp_processor.erl

```diff
@@ -1179,3 +1180,28 @@ additional_info(Key,
                 #proc_state{adapter_info =
                                 #amqp_adapter_info{additional_info = AddInfo}}) ->
     proplists:get_value(Key, AddInfo).
+
+parse_routing(Destination) ->
+    {Exchange0, RoutingKey} = rabbit_routing_util:parse_routing(Destination),
+    Exchange1 = maybe_apply_default_topic_exchange(Exchange0),
+    {Exchange1, RoutingKey}.
+
+maybe_apply_default_topic_exchange("amq.topic"=Exchange) ->
+    FromConfig = application:get_env(rabbitmq_stomp, default_topic_exchange, <<"amq.topic">>),
```

Comment:
This means we hit `application_controller` and a shared ETS table for every message published. I would like to extract this into the processor state since we can afford caching this value as it will be effectively a constant for 99.9% of users.

Reply:
On it! Thanks for noticing.